doc/api/stream.md

```diff
@@ -405,6 +405,9 @@ Is `true` after [`writable.destroy()`][writable-destroy] has been called.
 <!-- YAML
 added: v0.9.4
 changes:
+  - version: v13.11.0
+    pr-url: https://github.com/nodejs/node/pull/32238
```

Comment:
@rexagod If there’s a behavioral change, this should point to the URL of the PR where the actual changed happened… Can you use that instead?